source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -4351,6 +4351,148 @@ export class ColumnSet extends CardElementContainer {
     }
 }
 
+export class Source {
+    mimeType: string;
+    url: string;
+
+    parse(json: any): Source {
+        this.mimeType = json["mimeType"];
+        this.url = json["url"];
+        return this;
+    }
+
+    toJSON() {
+        return { mimeType: this.mimeType, url: this.url };
+    }
+}
+
+export class Media extends CardElement {
+    poster: string;
+    altText: string;
+    sources: Array<Source> = [];
+    private _mediaType: string;
+    private _mediaElement: HTMLMediaElement;
+    static readonly acceptedMediaTypes = ["audio", "video"];
+
+    private processSources(): Array<Source> {
+        let typeFilter = (t): boolean => {
+            if (Media.acceptedMediaTypes.indexOf(t)) {
+                this._mediaType = t;
+                typeFilter = (tt) => tt === this._mediaType;
+                return true;
```

Comment:
I usually put a blank line above a return statement if it is not the only (first) one in the block.

Reply:
Not a fan of vertical spaces, but this seems to bother many people (some good friends even mock me for this). I will follow the guidelines you're expressing here.